src/HttpClient.php

```diff
@@ -134,12 +179,14 @@ protected function setCurlOptions($handle, RequestInterface $request): void
             $queryUrl .= http_build_query(Request::buildHTTPCurlQuery($body));
```

Comment:
Why are you not using `$body["data"]` instead of `$body` here?
This will change the behavior of query params and break the existing applications